server/controllers/auth/login.js

```diff
@@ -45,11 +52,19 @@ const login = (req, res) => {
                 bcrypt.compare(password, data.password, (err, result) => {
                   if (!result) res.status(400).json(errorResponse);
                   else {
+                    console.log(data);
                     const userToken = { userId: data.id };
                     const cookie = sign(userToken, process.env.SECRET_KEY);
-                    res
-                      .cookie('admin', cookie, { httpOnly: true })
-                      .json({ status: 'successfully', role: 'admin', data });
+                    res.cookie('admin', cookie, { httpOnly: true }).json({
+                      status: 'successfully',
+                      role: 'admin',
+                      data: {
+                        email,
+                        fullName: `${data.fullName}`,
+                        location: `${data.location}`,
+                        avatar: `${data.avatar}`,
+                      },
```

Comment:
admin didn't have these data like applicant so you have to return only email & username 
```suggestion
                      data: {
                        email,
                        userName: `${data.userName}`,
                      },
```